build/root/WORKSPACE

```diff
@@ -73,7 +73,7 @@ docker_pull(
     digest = "sha256:fc1b461367730660ac5a40c1eb2d1b23221829acf8a892981c12361383b3742b",
```

Comment:
Please update this to `sha256:547c13f2ced76531600eb0a26051efb59e49a1f5d3522dda4028d2cba5a8b624`.